doc/installationguide.md

```diff
@@ -226,6 +226,12 @@ used for the same purpose. For instance:
     the IoTAgent runs in a single thread. For more details about multi-core functionality, please refer to the
     [Cluster](https://nodejs.org/api/cluster.html) module in Node.js and
     [this section](howto.md#iot-agent-in-multi-thread-mode) of the library documentation.
+-  **jsonLdContext**: the location of the NGSI-LD `@context` element which provides additional information allowing the computer to
```

Comment:
What is the difference with contextBroker.jsonLdContext field? Is not the same?

Reply:
The documentation of `contextBroker.jsonLdContext` was missed in the previous PR - I've just added it in this PR instead. This is not *another* new configuration attribute, just adding the missing documentation for the existing one.

Reply:
The same text is duplicated in L44-45. I'd suggest to avoid duplication and have it only in one place and that place should be L44-45 as it is the place closer to the configuration object in which jsonLdContext is being used (i.e. `contextBroker`).

Reply:
Fixed e099b84